channels/ON-CHAIN.md

```diff
@@ -252,26 +253,32 @@ Serialization defined [here](../serializations.md#channel-close-solo-transaction
 - `nonce`: taken from the `from`'s account
 - `fee`
 
-Proof of inclusion represents the internal channel. At the
-bare minimum it has to include all accounts and their balances but can also
-include contracts and contract calls. It must provide enough information to
+Proof of inclusion represents the internal channel' s state. At the
+bare minimum it has to include all accounts and their balances. It must provide enough information to
 close the channel. Miners are to check balances in it and use this data to initiate the
 procedure of channel solo closing.
+If there are any contracts in the channel and those have balances of their own - they are
+not provided in the proof of inclusion but they are rather to be force-pushed in
+subsequent transactions. It is up to participants to decide if they want to
+post them at all. Thus the accumulative balances of the accounts in
+the solo-close transaction can be lower than channel balance persisted
+on-chain.
 
 Payload is a valid transaction that has:
 * `state_hash` equal to the proof of inclusion's root hash. This is a proof
   that the PoI is a correct one
 * `channel_id` being the same as the transaction `channel_id`
-* `round` being greater or equal to the last on-chain `round` for that channel id
+* `round` being greater than the last on-chain `round` for that channel id. If
+  the last on-chain transaction had been produced by forced progress, chain had
+  been produced by a force progress transaction - the `round` can also be equal
+  to it. In this case the co-signed payload overwrites the on-chain produced
+  signle-signed force progress
```

Comment:
single-signed?

Reply:
Yes, the whole idea behind forcing progress is that one party can call a contract unilaterally. Please check [my comment above](https://github.com/aeternity/protocol/pull/180#discussion_r214031361)

Reply:
Thanks @velzevur, I'm beginning to get the idea. There is a typo in the doc, `s/signle/single/g`.